<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>570</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>초성 퀴즈 프로그램</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextBrowser" name="tb_content">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>43</y>
      <width>471</width>
      <height>471</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_username">
    <property name="geometry">
     <rect>
      <x>278</x>
      <y>13</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>6</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>이름 입력</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_connect">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>11</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>연결</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_inputText">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>523</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>12</x>
      <y>15</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>문제 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_question">
    <property name="geometry">
     <rect>
      <x>52</x>
      <y>15</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pb_tbClear">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>11</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>지우기</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_hostip">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>13</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>192.168.5.202</string>
    </property>
    <property name="maxLength">
     <number>15</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>호스트 IP</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="le_hostport">
    <property name="geometry">
     <rect>
      <x>234</x>
      <y>13</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>8083</string>
    </property>
    <property name="maxLength">
     <number>5</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>포트 입력</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>le_hostip</tabstop>
  <tabstop>le_hostport</tabstop>
  <tabstop>le_username</tabstop>
  <tabstop>pb_connect</tabstop>
  <tabstop>pb_tbClear</tabstop>
  <tabstop>tb_content</tabstop>
  <tabstop>le_inputText</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
